ovos_utils/file_utils.py

```diff
@@ -40,9 +43,13 @@ def get_temp_path(*args):
     return path
 
 
-def get_cache_directory(folder):
-    # optional import to use ram for cache
-    # does not work in windows!
+def get_cache_directory(folder: str) -> str:
+    """
+    Get a temporary cache directory, preferably in RAM.
+    This method is not compatible with Windows!
```

Comment:
it is compatible with Windows, but wont use the RAM in that case, it has a platform check below